Tests/MongoSwiftSyncTests/SDAMMonitoringTests.swift

```diff
@@ -51,52 +47,48 @@ final class SDAMTests: MongoSwiftTestCase {
         }
         let hostAddress = try ServerAddress(host)
 
-        // check event count and that events are of the expected types
-        expect(receivedEvents.count).to(beGreaterThanOrEqualTo(5))
```

Comment:
in this test we previously saw:
- A topology opening event
- A topology changed event (change from unknown to single, no hosts in list)
- A server opening event
- A server changed event, from unknown to standalone
- A topology changed event, adding in the new server to the topology

We now see:
- A topology opening event
- A server opening event
- A server changed event, from unknown to standalone
- A topology changed event, from unknown to single

Previously in this case libmongoc by default would behave like the directConnection=true option was specified, which meant the topology would be assumed to be single right off the bat before discovering servers. Now that we specify directConnection=false that transition doesn't happen until the server has been discovered.